docs/astro/src/content/docs/reference/window/window.mdx

```diff
@@ -67,8 +67,7 @@ Place a single `MenuBar` element into a `Window` to declare a menu bar for the w
 There can only be one `MenuBar` element in a `Window` and it must not be in a `for` or a `if`.
 :::
 
-The `MenuBar` doesn't have properties, but it should contain <Link type="MenuItem" /> as children that represent top level entries in the menu bar.
-Nest `MenuItem` elements to declare sub-menus.
+The `MenuBar` doesn't have properties, but it should contain <Link type="Menu" /> as children that represent top level entries in the menu bar.
```

Comment:
I'm tempted to suggest this:

```suggestion
The `MenuBar` doesn't have properties, but it must contain <Link type="Menu" /> as children that represent top level entries in the menu bar.
```

I know it's not enforced, but... does a `MenuBar` without `Menu` make *any* sense at all?